cpp/include/DataStorm/Node.h

```diff
@@ -27,86 +27,54 @@ namespace DataStorm
     {
     public:
         /// Constructs a DataStorm node.
-        /// A node is the main DataStorm object. It is required to construct topics.
-        /// @param argc The number of arguments in argv.
-        /// @param argv The configuration arguments.
-        /// @param configFile The path to an optional Ice configuration file.
+        /// A node is the main DataStorm object. It is required to construct topics. The node uses the given Ice
```

Comment:
This needs more refactoring. I think the correct solution is to introduce a NodeOptions struct used by the primary constructor. And all other constructors are convenience constructors.